build/release.sh

```diff
@@ -32,10 +32,19 @@ if ! git_user="$(git config --get user.email)"; then
   exit 1
 fi
 
-# Build the release.
 export BUILD_USER="$git_user"
 export BUILD_DATE=$( date +%Y%m%d ) # Release date is only to day-granularity
 export VERBOSE=true
+
+# Get and build libpfm4 for cAdvisor with perf support
+wget https://sourceforge.net/projects/perfmon2/files/libpfm4/libpfm-4.10.1.tar.gz
```

Comment:
I don't think we need to install libpfm4 as part of the release script.  The documentation in development/build.md should be sufficient.  I mostly just wanted to have libpfm used as to go flag for the docker build, which you added below.